packages/elements/src/overlay/__test__/managers/focus-manager.test.js

```diff
@@ -82,7 +82,8 @@ describe('overlay/manager/FocusManager', () => {
           const activeElement = document.activeElement;
 
           element.opened = false;
-          await openedUpdated(element);
+          await elementUpdated(element);
+          await nextFrame(); // Must be only one frame
```

Comment:
why is must be one frame? compare to previously it used openedUpdated that use 3 frames.

Reply:
- This fixing is for this test case only and it does not affect others.
- The reason why it must be one frame is the `Web Test Runner` injects some `<script />` to the body on the fly, which will affect the `activeElement` this fix tries to get the `activeElement` before injection.